src/Command/DumpConfigCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command;
+
+use Nette\Neon\Neon;
+use PHPStan\ShouldNotHappenException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use function is_array;
+use function is_string;
+
+class DumpConfigCommand extends Command
```

Comment:
I agree, it's simple and I'd do it myself if I prioritized it :)

It only dump parameters so the command class and the command name should reflect that.